hack/build/build-push-multiarch-images.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+#Copyright 2025 The AAQ Authors.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+################################################################################
+# This script builds and push multi-arch manifest images
+# The reason we can't split the build and the push, as desired, is that both
+# docker and earlier versions of podman, cannot build manifests from local
+# images, and so we must push the per-architecture images first, before creating
+# the manifest from them. This is fixed in podman 5, but many build machine are
+# still using older versions.
+################################################################################
+
+set -e
+
+script_dir="$(readlink -f $(dirname $0))"
+source "${script_dir}/common.sh"
+source "${script_dir}/config.sh"
+
+if ! command -v docker &> /dev/null && ! command -v podman &> /dev/null; then
+    echo "Error: Neither Docker nor Podman found. Please install one of them."
+    exit 1
+fi
+
+cri_cmd=""
+insecure=""
+if command -v podman &> /dev/null; then
+    cri_cmd="podman"
+    insecure="--tls-verify=false"
```

Comment:
Not sure what the purpose of the `--tls-verify=false` is, but i don't think it is needed anymore.
Overall i would drop this whole codeblock and instead use `AAQ_CRI` already sourced from `common.sh`